docs/demos/UAV_tutorial.py

```diff
@@ -0,0 +1,279 @@
+# elevation measurements (from a given radar position). We will use Stone Soup's simple
+# :class:`~.SingleTargetTracker` to perform the tracking. At this point we are primarly interested
+# in the necessary motion models that may be needed so the example is fairly simple, but
+# we want to be able to easily expand the simulation to handle more complex scenarios.
+#
+# Items to note:
+#
+# - Assumes a single target track, which simplifies track management.
+# - There is no clutter, and no missed detections. So 1:1 Data Association.
+# - Need an initiator and deleter for the tracker.
+# - GPS updates are 1 sec., we assume radar revisit is the same (little unrealistic)
+#
+# We are assuming a ground based radar:
+#
+# - Radar has course elevation resolution and fine bearing resolution.
+# - Use range standard deviation of 3.14 m as a replacement for range resolution.
+
+
+
+# %%
+# Setup: transition model, measurement model, updater and predictor
+# -----------------------------------------------------------------
+
+import numpy as np
+from stonesoup.models.transition.linear import (
+    ConstantVelocity,
+    CombinedLinearGaussianTransitionModel
+    )
+from stonesoup.predictor.kalman import ExtendedKalmanPredictor
+from stonesoup.updater.kalman import ExtendedKalmanUpdater
+from stonesoup.models.measurement.nonlinear import (
+    CartesianToElevationBearingRange
+    )
+from stonesoup.types.array import CovarianceMatrix
+
+
+transition_model = CombinedLinearGaussianTransitionModel(
+    [ConstantVelocity(1.0),
+     ConstantVelocity(1.0),
+     ConstantVelocity(1.0)])
+
+# Model coords = elev, bearing, range. Angles in radians
+meas_covar = np.diag([np.radians(np.sqrt(10.0))**2,
+                      np.radians(0.6)**2,
+                      3.14**2])
+
+meas_covar_trk = CovarianceMatrix(1.0*meas_covar)
+meas_model = CartesianToElevationBearingRange(
+            ndim_state=6,
+            mapping=np.array([0, 2, 4]),
+            noise_covar=meas_covar_trk)
+predictor = ExtendedKalmanPredictor(transition_model)
+updater = ExtendedKalmanUpdater(measurement_model=meas_model)
+
+# %%
+# Setup CSV reader & feeder
+# -------------------------
+# This part uses 2 Stone Soup detector type of classes:
+#
+# - :class:`~.CSVGroundTruthReader` - reads our CSV file which contains: timestamp,
+#   latitude, longitude, altitude and other miscellaneous data.
+# - :class:`~.LLAtoENUConverter` - this is a feeder, but it converts our lat, long, 
+#   alt data into Cartesian (ENU).
+#
+# The Cartesian(ENU) data will be converted to Range, Bearing, Altitude later on.
+# More involved detector could:
+#
+# - Add clutter.
+# - Handle :math:`P_d` behaviour. Could be based on radial velocity or other
+#   parameters.
+# - Handle radar revisit times.
+# - Add unknown number & multiple targets.
+#
+from stonesoup.reader.generic import CSVGroundTruthReader
+
+ground_truth_reader = CSVGroundTruthReader(
+    path='UAV_Rot.csv',
```

Comment:
Need to provide a download link to the data. You can see example here using the `:download:` directive: https://github.com/dstl/Stone-Soup/blob/b6df2598989b1d03e27a6ab47e64897ba566b6d2/docs/demos/AIS_Solent_Tracker.py#L20-L31

Path should be the same as AIS demo CSV, so just need to change filename to `UAV_Rot.csv`. Sphinx will keep sure this is packaged up with documentation then.